### PR TITLE
Create a test to check the issue #4501

### PR DESCRIPTION
**Pull request by paulovitin**
Adds:
  - Check if after the user confirms the email and change the email again a new verification email is sent

### CODE REVIEW

**Comment by montymxb** [comment]
Oh...well ignore what I was saying before. Didn't realize this was already up, totally missed that 🤣 . I'll suggest some changes here.

File: spec/EmailVerificationToken.spec.js [hunk 1]
```diff
@@ -777,4 +777,63 @@ describe("Email Verification Token Expiration: ", () => {
       });
   });
 
+  it('When a user has verified their email, and then change it, the field emailVerified should be false and the new link will be send to the user', done => {
```
Comment (montymxb) [inline spec/EmailVerificationToken.spec.js]: A suggestion to reword this to be a bit shorter
> emailVerified should be set to false after changing from an already verified email

Comment (paulovitin) [reply spec/EmailVerificationToken.spec.js]: Man, do you speak a lot hahaha.
